packages/sanity/src/_internal/cli/commands/backup/backupGroup.ts

```diff
@@ -0,0 +1,26 @@
+import {CliCommandGroupDefinition} from '@sanity/cli'
+
+// defaultApiVersion is the backend API version used for dataset backup.
+export const defaultApiVersion = 'vX'
+
+const datasetBackupGroup: CliCommandGroupDefinition = {
+  name: 'backup',
+  signature: '[COMMAND]',
+  description: 'Manage dataset backups.',
+  isGroupRoot: true,
+  hideFromHelp: false,
```

Comment:
I think we do want to hide this for now don't we?

Reply:
Agree. Set `hideFromHelp: true` now.